### PR TITLE
381 Allow multiple contact emails for a sponsor

### PR DESCRIPTION
**Pull request by dpoirier**
Must be at least one

Fixes: #381

### CODE REVIEW

File: pycon/sponsorship/admin.py [hunk 1]
```diff
@@ -58,7 +60,9 @@ class SponsorAdmin(admin.ModelAdmin):
     readonly_fields = ["approval_time"]
 
     def contact(self, sponsor):
-        return mark_safe('<a href="mailto:%s">%s</a>' % (escape(sponsor.contact_email), escape(sponsor.contact_name)))
+        # comma-separated emails in mailto: should work: https://www.ietf.org/rfc/rfc2368.txt
+        # but the commas need to be URL-quoted
+        return mark_safe('<a href="mailto:%s">%s</a>' % (escape(quote(','.join(sponsor.contact_emails))), escape(sponsor.contact_name)))
```
Comment (rebecca-caktus) [inline pycon/sponsorship/admin.py]: You can use `format_html`: https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.html.format_html

File: pycon/sponsorship/forms.py [hunk 2]
```diff
@@ -32,13 +36,15 @@ def save(self, commit=True):
 
 
 class SponsorDetailsForm(forms.ModelForm):
+    contact_emails = MultiEmailField()
```
Comment (rebecca-caktus) [inline pycon/sponsorship/forms.py]: Should the same help text (one email per line) be added here as above?

**Comment by rebecca-caktus** [comment]
:email: :email:

File: pycon/sponsorship/forms.py (hunk 2 above)
Comment (dpoirier) [inline pycon/sponsorship/forms.py]: On Mon, Jul 20, 2015 at 2:18 PM, Rebecca <notifications@github.com> wrote:

> Should the same help text (one email per line) be added here as above?


​Good catch!​

File: pycon/sponsorship/admin.py (hunk 1 above)
Comment (dpoirier) [inline pycon/sponsorship/admin.py]: On Mon, Jul 20, 2015 at 2:15 PM, Rebecca <notifications@github.com> wrote:

> You can use format_html
>

​I've updated this if you want to take another look.​

**Pull request merged**